### PR TITLE
1064 geometry snapper

### PR DESCRIPTION
**Pull request by yjacolin**

### CODE REVIEW

File: source/docs/user_manual/plugins/plugins_geometry_snapper.rst [hunk 1]
```diff
@@ -1,10 +1,27 @@
 |updatedisclaimer|
 
+.. index:: Geometry, Topology, Snapping, Plugin, Geometry snapper plugin
+
 .. _geometry_snapper:
 
 Geometry Snapper Plugin
 =======================
 
+The Geometry Snapper tool allow align the edges and vertices of one vector
```
Comment (DelazJ) [inline source/docs/user_manual/plugins/plugins_geometry_snapper.rst]: allow**s to** align **automatically** (to let the user understand that it's more than the snapping option; it's not a manual process)

File: source/docs/user_manual/plugins/plugins_geometry_snapper.rst [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+layer to the edges and vertices of a second layer using a user defined
+tolerance.
+
+The below interface shows the settings of this plugin. User needs to choose the
+layer to change (see :guilabel:`input vector layer`) and the :guilabel:`reference
+layers` to snap to. A :guilabel:`Maximum snapping distance (map units)` allows
+to change the snapping tolerance.
+
+You can snap only selected feature checking the |checkbox| :guilabel:`Only
```
Comment (DelazJ) [inline source/docs/user_manual/plugins/plugins_geometry_snapper.rst]: feature(s)?

**Comment by DelazJ** [comment]
few comments. LGTM

**Comment by yjacolin** [comment]
I will be offline from now until sunday. Feel free to merge it :) See you next monday for the Doc release week :)

**Pull request merged**